docs/workflows/user-provisioning.md

```diff
@@ -0,0 +1,155 @@
+---
+title: User Provisioning
+sidebar_position: 7
+
+# Display h2 to h2 headings
+toc_min_heading_level: 2
+toc_max_heading_level: 2
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# User Provisioning Overview
+
+Before creating a Passkey, a user must be provisioned as an identity under a realm. An identity is a unique identifier that may be used by an end-user to gain access governed by Beyond Identity. An end-user may have multiple identities. A realm can have many identities.
+
+Identities can be part of a group. A realm can have many groups. An identity can belong to many groups. Groups are commonly used as a predicate in a policy rule. (e.g. Allow group "Beyond Identity Administrators" to access the "Beyond Identity Admin Console" application)
+
+You can provision a user with:
+
+1. [Beyond Identity Admin Console](user-provisioning#provision-user-with-the-beyond-identity-admin-console)
+2. [Beyond Identity APIs](user-provisioning#provision-user-with-the-beyond-identity-api)
+3. [SCIM](../scim/v1/scimv1)
+
+## Provision User with the Beyond Identity Admin Console
+
+From the Beyond Identity Admin Console you can add identities and manage groups.
+
+### Identities
+
+The simplest way to create an identity is through the Beyond Identity Admin Console. First select a realm. Under the "PROJECT MANAGEMENT" tab, select "Identities", and then click on "Add identity". From there you can enter the user's name, username, and email.
+
+![Identities](./screenshots/user-provisioning-identities.png)
+
+![Add Identity](./screenshots/user-provisioning-add-identity.png)
+
+### Groups
+
+You can manage groups in the Beyond Identity Admin Console, creating groups, adding to and removing identities from groups. Under the "PROJECT MANAGEMENT" tab, select "Groups".
+
+![Groups](./screenshots/user-provisioning-groups.png)
+
+## Provision User with the Beyond Identity API
+
+Before making any API calls you'll want to generate an `accessToken`. Check out [Create an API token](./api-token) for help creating an access token.
+
+The following examples show how to create an identity via API. For deleting, updating, and managing groups via API, see the [API documentation](https://developer.beyondidentity.com/api/v1).
+
+<Tabs groupId="user-provisioning-api-platform" queryString>
+  <TabItem value="curl" label="Curl">
+
+```bash title="/identities"
+curl https://api-$REGION.beyondidentity.com/v1/tenants/$TENANT_ID/realms/$REALM_ID/identities \
+-X POST \
+   -H "Authorization: Bearer $API_TOKEN" \
+   -H 'Content-Type: application/json' \
+   -d '{"identity":{"display_name":"NAME","traits": {"type": "traits_v0","username": "USERNAME",
+"primary_email_address":"EMAIL"}}}'
+```
+
+  </TabItem>
+  <TabItem value="node" label="Node">
+
+```jsx title="/identities"
+const identityResponse = await fetch(
+  `https://api-${REGION}.beyondidentity.com/v1/tenants/${TENANT_ID}/realms/${REALM_ID}/identities`,
```

Comment:
Should the examples below also use the `${}` notation for external/environment variables?

Reply:
ah I was using https://curlconverter.com for the code snippets. If `${}` works for the other languages I can update too

Reply:
Ah, got it! `${}` definitely doesn't work out of the box in the other languages but I think it's a little more clear that those values are meant to be replaced somehow. Otherwise, if we're really trying to provide comprehensive code snippets, we could add environment variable handling into the snippets below.

Reply:
Hm what do you mean "add environment variable handling into the snippets below"?

Reply:
oh ok I see what you mean. Yea I think it's fine if it doesn't compile. It'll help point to the pieces they need to fill in.

Reply:
Got it, yeah in that case I think using the `${}` for all such values/variables makes sense!

Reply:
I may come back with another MR for this since we're doing this in a lot of places.